examples/advection_2d_square/setrun.py

```diff
@@ -113,8 +113,8 @@ def setrun(claw_pkg='amrclaw'):
     if clawdata.output_style==1:
         # Output ntimes frames at equally spaced times up to tfinal:
         # Can specify num_output_times = 0 for no output
-        clawdata.num_output_times = 10
-        clawdata.tfinal = 2.0
+        clawdata.num_output_times = 1
+        clawdata.tfinal = 0.2
```

Comment:
Did you mean to change these?

Reply:
I reduced it to easily try out different `memsize` values, but plan to clean it up again before merging.